fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsMathUtil.java

```diff
@@ -59,4 +64,21 @@ public static double divide(double a, double b) {
         return a / nonZeroDivisor(b);
     }
 
+    /**
+     * compute the multi columns unite ndv
+     */
+    public static double multiNdv(List<Double> ndvs) {
```

Comment:
how about rename to accumulativeNdv or jointNdv ?

Reply:
done